### DISCUSSION TITLE
关于顶栏相关的四个建议：

### DISCUSSION THREAD

**Discussion by DD224ever** [the1812/Bilibili-Evolved#4923] [功能建议 / Ideas]
1，如题，看我下头，我们插件的B站LOGO图标鼠标悬浮上去是没有任何效果的，有点浪费了，反而还有多出来“排行”和“主站”两个模块，大部分用户是更喜欢简洁些的，希望在设置加个“LOGO图标悬浮显示选择：排行、分区等选项”，可以看看另一个插件的效果：
![image_95](https://github.com/user-attachments/assets/56ff17c6-5128-4fbd-a377-11ed21799c93)
![image_94](https://github.com/user-attachments/assets/fbc3dc9a-9e05-4f9b-bde9-c83c788053de)
2，自定义顶栏的透明度能不能给个调整条，“透明填充”的透明程度对我来说不太够，
3，另外希望像之前一样可以把自定义顶栏布局的入口放到“功能”页面：
![image_93](https://github.com/user-attachments/assets/b11f38bb-b8f6-4493-88ca-fcd76050852a)

**Comment by DD224ever** [comment]
图一图三放反了，懂我意思就好。

**Reply by Geeyun-JY3** [reply]
你可以编辑的。

**Comment by DD224ever** [comment]
加个4吧：希望Bilibili-Evolved的分区分类入口也能像图三BewlyBewly那么全，然后布局的话两列太长图三又太宽，作者自己把握一下宽度然后放成三列最好，图三这种透明度能给到我们的自定义顶栏就好了。

除了LOGO没有分区入口以及透明度不够，我们顶栏还是比BewlyBewly好用很多的，好在他兼容我们的顶栏，两个插件一起用我觉得我强的可怕。

**Reply by Geeyun-JY3** [reply]
关于分区入口一直以来的思路是跟随原版顶栏：https://github.com/the1812/Bilibili-Evolved/discussions/1511#discussioncomment-9649865

**Comment by Geeyun-JY3** [comment]
3 已经做了，请善用搜索：#2666
https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.9.2-preview
正式版也已实装：https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.9.3

**Comment by Geeyun-JY3** [comment]
1. 破站现在的原版顶栏分区菜单的光标悬停触发范围倒是覆盖了 bilibili logo 和`首页`：
![image](https://github.com/user-attachments/assets/421a5172-7fbb-415b-bdb8-a65ed9a34cf2)
我不太记得`自定义顶栏`仿的这个时期的原版顶栏中的`主站`（通过脚本 [Bilibili-Old](https://github.com/MotooriKashin/Bilibili-Old/) 能看到当年原版顶栏的`主站`二字左边只有一个小小的 favicon）有没有一个光标悬停出来分区菜单了（所以这个分区菜单入口的加入是因为 #100？或者可以问问 @the1812。不过看 [1.8.0 的更新日志](https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.8.0)应该是可以确定原版顶栏是没有的），既然光标悬停在现在的原版顶栏的 bilibili logo 和`首页`都能出来分区菜单，或者可以考虑给选项让 bilibili logo 也能光标悬停触发分区菜单？

***

（放些我写这条评论找到的东西）

- 本脚本早期优化后的顶栏（我不确定**顶栏效果**那两张图是原版顶栏还是早期未正式发布的`自定义顶栏`还是两者各一张）：https://github.com/the1812/Bilibili-Evolved/tree/59eaffcb5258fd25606453c3ea8418db5e70baed#%E6%A0%B7%E5%BC%8F%E8%B0%83%E6%95%B4
- 本脚本的`自定义顶栏`刚出来的样子：https://github.com/the1812/Bilibili-Evolved/tree/c941a5d4e576d6814d88706e5d66f11e13540ef9#%E8%87%AA%E5%AE%9A%E4%B9%89%E9%A1%B6%E6%A0%8F
